com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -143,6 +143,10 @@ public partial class UniversalRenderPipelineAsset : RenderPipelineAsset, ISerial
         [SerializeField] int m_AdditionalLightsShadowResolutionTierMedium = 512;
         [SerializeField] int m_AdditionalLightsShadowResolutionTierHigh = 1024;
 
+        // Reflection Probes
```

Comment:
Do we want these startup values?
What are others' opinions?

Reply:
Keeping the default as false will provide a clean upgrade process - it will look the same as it does in current URP. It does make it harder to discover the feature as users will have to actively enable it first.

Reply:
You could do it as an upgrade, such that new assets have the feature enabled, and old assets get upgraded such that the feature is disabled.

Reply:
Question is if we want all new assets to have feature enabled or not? For box projection definitely not, for blending depends on user decision, quality or project. That's why I think we really should have something to ask user what is the quality intended when he is creating a new asset and create settings based on those quality presets.

Reply:
Having both disabled by default poses the least amount of risk right now. A future PR will set up suitable defaults for every project. For the time being users will have to opt-in.